test/Microsoft.AspNetCore.SignalR.Client.Tests/ServerSentEventsParserTests.cs

```diff
@@ -110,31 +110,31 @@ public void ParseSSEMessageIncompleteParseResult(string encodedMessage)
             var parser = new ServerSentEventsMessageParser();
             var pipe = new Pipe();
 
-                byte[] message = null;
-                SequencePosition consumed = default, examined = default;
+            byte[] message = null;
```

Comment:
I Ctrl-K+Ctrl-D'd most of the files I cleaned up and things like this are what happened as a result :)

Reply:
You should do that when you code normally.

Reply:
![https://media.giphy.com/media/A6H1A9rhetsXK/giphy.gif](https://media.giphy.com/media/A6H1A9rhetsXK/giphy.gif)